Class Presentations/reveal-presentations/ObjectsAndDom.html

```diff
@@ -0,0 +1,696 @@
+					</section>
+					<section>
+						<p> What is the default label in a switch statement for? </p>
+					</section>
+					<section>
+						<p> The default label creates an option for when the variable does not match any of the cases.</p>
+					</section>
+					<section>
+						<p> Write a switch statement for a traffic light. </p>
+					</section>
+					<section data-markdown>
+						(Example)
+						```javascript
+						var color = "green";
+						switch (color) {
+							case "green":
+								speed = 35;
+								break;
+							case "yellow":
+								speed = 45;
+								break;
+							case "red":
+								speed = 0;
+								break;
+							default:
+								speed = 10;
+						}
+						```
+						</section>
+					<section>
+						<p> What is a for loop?</p>
+					</section>
+					<section>
+						<p> A for loop is a control structure that allows a statement to be
+							 repeated until a certain condition is met.</p>
+					</section>
+					<section>
+						<p> Let's write a for-loop that tells someone to clap ten times.</p>
+					</section>
+					<section data-markdown>
+						(Example)
+						```javascript
+						for(var i=0; i<10; i++) {
+							clap();
+						}
+						```
+					</section>
+					<section>
+						<p> What is a while loop?</p>
+					</section>
+					<section>
+						<p> A while loop is a control structure that allows a statement to be
+							 repeated while a certain condition is under effect. The condition
+						 is checked at the beginning of each loop.</p>
+					</section>
+					<section>
+						<p> Let's write a while-loop that tells someone to clap ten times.</p>
+					</section>
+					<section data-markdown>
+						(Example)
+						```javascript
+						var i = 0;
+						while(i<10) {
+							clap();
+							i++;
+						}
+						```
+					</section>
+					<section>
+						<p> What is a do-while loop? (How does it differ from a while loop?)</p>
+					</section>
+					<section>
+						<p> A do-while loop is a control structure that allows a statement to be
+							 repeated while a certain condition is under effect. However, the code within
+						 a do-while looop is always executed at least once, and the condition is
+						 checked at the end of each loop.</p>
+					</section>
+					<section>
+						<p> Let's write a do-while loop that tells someone to clap ten times.</p>
+					</section>
+					<section data-markdown>
+						(Example)
+						```javascript
+						var i = 0;
+						do {
+							clap();
+							i++;
+						} while (i < 11);
+						```
+						or
+						```javascript
+						var i = 1;
+						do {
+							clap();
+							i++;
+						} while (i <= 10);
+						```
+						(many different ways to do this)
+					</section>
+				</section>
+
+				<section>
+					<h2>What are objects?</h2>
+				</section>
+
+			<section>
+				<h2>Objects</h2>
+				<p>"A collection of named values"</p>
+				<p>"Variables containing variables"</p>
+				<p>"The building blocks of JavaScript"</p>
```

Comment:
I am not sure if it would help or confuse them at this point to point out that everything in JS is an object, but it could be worth going into later on? Use your best judgment.